feature_engine/encoding/decision_tree.py

```diff
@@ -199,7 +199,6 @@ def __init__(
         variables: Union[None, int, str, List[Union[str, int]]] = None,
         ignore_format: bool = False,
     ) -> None:
-
```

Comment:
this file was probably modified with black, but it is not really part of this PR. Could we try and remove it please?